src/scc_hypervisor_collector/cli/__init__.py

```diff
@@ -33,9 +34,19 @@ def create_logger(level: str,
     formatter = logging.Formatter(fmt_str)
     loghandler: Any = None
     if logfile:
-        loghandler = PermissionsRotatingFileHandler(logfile,
-                                                    maxBytes=(0x100000 * 5),
-                                                    backupCount=5)
+        try:
+            loghandler = \
```

Comment:
Stylistic suggestion: you could just pull the `PermissionsRotatingFileHandler(`on to the end of this line and avoid the need to use a `\` at all.